tests/py_test.py

```diff
@@ -234,3 +237,217 @@ def test_save_mp4(db):
     f.close()
     table.columns('frame').save_mp4(f.name)
     run(['rm', '-rf', f.name])
+
+@pytest.fixture()
```

Comment:
We should probably start to break out tests into separate files, e.g. `fault_tolerance_test.py` or something.